test/unit/specs/stores/filter-store.spec.js

```diff
@@ -0,0 +1,464 @@
+import { setActivePinia, createPinia } from 'pinia'
```

Comment:
Can we rename this file to `search-store.spec.js` to match the module it tests? I guess it really would just be `stores/search.spec.js`, either way though.